src/pages/students.js

```diff
@@ -0,0 +1,346 @@
+              <h4>500 GB/month of Data Ingest</h4>
+              <p>
+                Collect, analyze, and alert on all your metrics, events, logs,
+                and traces from any source
+              </p>
+            </div>
+            <div className={styles.point}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="cpu"
+                size="4rem"
+              />
+              <h4>Free Tier AI</h4>
+              <p>
+                Instantly detect, diagnose, and resolve issues before they
+                become a problem
+              </p>
+            </div>
+            <Button
+              as={ExternalLink}
+              className={styles.nominateButton}
+              href="https://forms.gle/Zkdub5e1x4MNqSKW9"
+              variant={Button.VARIANT.PRIMARY}
+            >
+              Sign up for a free account{' '}
+            </Button>
+          </section>
+          <section className={styles.faqSection}>
+            <h2>Frequently Asked Questions</h2>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                How do I access New Relic Student Edition?
+              </p>
+              <div className={styles.answer}>
+                <p>
+                  New Relic offers the Student Edition exclusively through the
+                  GitHub Student Pack. To verify your account's eligibility,
+                  please authorize your Github account.
+                </p>
+                <p>
+                  Learn more about GitHub Education if you don't have GitHub
+                  Student Developer Pack yet.
+                </p>
+                <ul>
+                  <li>
+                    Sign up up for Student Edition - you will be prompted to
+                    create or log in to your GitHub account
+                  </li>
+                  <li>
+                    Apply for the GitHub Student Developer Pack, which requires
+                    proof of enrollment in an accredited institution.
+                  </li>
+                  <li>
+                    Once you’re verified through GitHub, sign in to Student
+                    Edition with the token emailed to you to get started.{' '}
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>Can I purchase Student Edition?</p>
+              <p className={styles.answer}>
+                New Relic offers the Student Edition for FREE to students and
+                educators.
+              </p>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                Can I share my Student Edition with friends?
+              </p>
+              <p className={styles.answer}>
+                The student edition comes with 3 FSO users. Your friends and
```

Comment:
I don't think we should use "FSO" users here. That's not a clear public term. Also, as three is a small number, we should stick with the word:

> The student edition comes with three full-stack observability users...

Or even:

> The student edition comes with three [full](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type) users...